media-libs/lib3mf/files/lib3mf-2.2.0-include-cstdint.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/Source/Common/Platform/NMR_ImportStream_Unique_Memory.cpp b/Source/Common/Platform/NMR_ImportStream_Unique_Memory.cpp
```

Comment:
In future, include the Gentoo bug link + PR link in the patch itself at the top as well in case other distros pick it up, so they have provenance.

Reply:
Will do!